sonar-project.properties

```diff
@@ -9,7 +9,7 @@ sonar.projectVersion=1.0
 # This property is optional if sonar.modules is set.
 sonar.sources=src
 
-sonar.exclusions=.build, src/models/**, src/config/**, src/services/LambdaService.ts, src/services/S3BucketService.ts, src/handler.ts
+sonar.exclusions=.build, src/models/**, src/config/**, src/services/LambdaService.ts, src/handler.ts, src/services/SharePointAuthenticationService.ts, src/services/SharePointService.ts
```

Comment:
Not a fan of just excluding stuff for the sake of it. Good that they're so simple as to not need hardly any testing, but should add what little we can.

Reply:
Removed the services from exclusions